<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HR4HVE7" failures="0" tests="1" name="Test" time="1.292" errors="0" timestamp="2022-04-16T17:02:39 IST">
  <testcase classname="com.e2e.testcases.BaseTest" name="@BeforeMethod SetUpTest" time="1.185">
    <failure type="org.openqa.selenium.SessionNotCreatedException" message="Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder &amp;apos;C:\Users\Sathish\AppData\Local\Android\Sdk\platform-tools;C:\Users\Sathish\AppData\Local\Android\Sdk;C:\Users\Sathish\AppData\Local\Android\Sdk\tools\bin;&amp;apos; does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;DESKTOP-HR4HVE7&amp;apos;, ip: &amp;apos;192.168.0.115&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.2&amp;apos;
Driver info: driver.version: Driver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder &amp;apos;C:\Users\Sathish\AppData\Local\Android\Sdk\platform-tools;C:\Users\Sathish\AppData\Local\Android\Sdk;C:\Users\Sathish\AppData\Local\Android\Sdk\tools\bin;&amp;apos; does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
    at getResponseForW3CError (C:\Users\Sathish\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\Sathish\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;DESKTOP-HR4HVE7&amp;apos;, ip: &amp;apos;192.168.0.115&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.2&amp;apos;
Driver info: driver.version: Driver">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\Sathish\AppData\Local\Android\Sdk\platform-tools;C:\Users\Sathish\AppData\Local\Android\Sdk;C:\Users\Sathish\AppData\Local\Android\Sdk\tools\bin;' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-HR4HVE7', ip: '192.168.0.115', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: driver.version: Driver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\Sathish\AppData\Local\Android\Sdk\platform-tools;C:\Users\Sathish\AppData\Local\Android\Sdk;C:\Users\Sathish\AppData\Local\Android\Sdk\tools\bin;' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
    at getResponseForW3CError (C:\Users\Sathish\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\Sathish\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-HR4HVE7', ip: '192.168.0.115', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: driver.version: Driver
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:247)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:338)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:89)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:99)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:96)
at com.e2e.driver.EmulatorDriverImpl.getDriver(EmulatorDriverImpl.java:24)
at com.e2e.driver.DriverFactory.get(DriverFactory.java:18)
at com.e2e.driver.Driver.initDriver(Driver.java:27)
at com.e2e.testcases.BaseTest.SetUpTest(BaseTest.java:25)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)
... 36 more
Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\Sathish\AppData\Local\Android\Sdk\platform-tools;C:\Users\Sathish\AppData\Local\Android\Sdk;C:\Users\Sathish\AppData\Local\Android\Sdk\tools\bin;' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-HR4HVE7', ip: '192.168.0.115', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: driver.version: Driver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\Sathish\AppData\Local\Android\Sdk\platform-tools;C:\Users\Sathish\AppData\Local\Android\Sdk;C:\Users\Sathish\AppData\Local\Android\Sdk\tools\bin;' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
    at getResponseForW3CError (C:\Users\Sathish\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\Sathish\AppData\Local\Programs\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:1002)
at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
... 41 more
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod SetUpTest -->
  <testcase classname="com.e2e.testcases.BaseTest" name="@AfterMethod tearDownTest" time="-1.650108759163E9">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDownTest -->
  <testcase classname="com.e2e.testcases.FirstTest" name="tapAction" time="0.0">
    <skipped/>
  </testcase> <!-- tapAction -->
</testsuite> <!-- Test -->
